test/JDBC/expected/string_agg-vu-verify.out

```diff
@@ -0,0 +1,560 @@
+-- expression as column
+SELECT STRING_AGG(a,'-') FROM string_agg_t
+GO
+~~START~~
+text
+c-b-g-a-e-d-h
+~~END~~
+
+
+SELECT STRING_AGG(a,'-') FROM string_agg_t GROUP BY g
+GO
+~~START~~
+text
+b-g-e
+c-a-d-h
+~~END~~
+
+
+SELECT STRING_AGG(a,'-') WITHIN GROUP (ORDER BY sbid) FROM string_agg_t GROUP BY g
+GO
+~~START~~
+text
+h-d-c-a
+e-g-b
+~~END~~
+
+
+SELECT STRING_AGG(a,'-') WITHIN GROUP (ORDER BY sbid ASC) FROM string_agg_t GROUP BY g
+GO
+~~START~~
+text
+h-d-c-a
+e-g-b
+~~END~~
+
+
+SELECT STRING_AGG(a,'-') WITHIN GROUP (ORDER BY sbid DESC) FROM string_agg_t GROUP BY g
+GO
+~~START~~
+text
+a-c-d-h
+b-g-e
+~~END~~
+
+
+SELECT STRING_AGG(a,'-') WITHIN GROUP (ORDER BY id, sbid ASC) FROM string_agg_t GROUP BY g
+GO
+~~START~~
+text
+a-c-h-d
+g-b-e
+~~END~~
+
+
+SELECT STRING_AGG(a,'-') WITHIN GROUP (ORDER BY id, sbid DESC) FROM string_agg_t GROUP BY g
+GO
+~~START~~
+text
+a-c-d-h
+b-g-e
+~~END~~
+
+
+-- expression as expression of multiple columns
+SELECT STRING_AGG(a+b,'-') FROM string_agg_t
+GO
+~~START~~
+text
+cx-by-gu-az-ev-dw-ht
+~~END~~
+
+
+SELECT STRING_AGG(a+b,'-') FROM string_agg_t GROUP BY g
+GO
+~~START~~
+text
+by-gu-ev
+cx-az-dw-ht
+~~END~~
+
+
+SELECT STRING_AGG(a+b,'-') WITHIN GROUP (ORDER BY sbid) FROM string_agg_t GROUP BY g
+GO
+~~START~~
+text
+ht-dw-cx-az
+ev-gu-by
+~~END~~
+
+
+SELECT STRING_AGG(a+b,'-') WITHIN GROUP (ORDER BY sbid ASC) FROM string_agg_t GROUP BY g
+GO
+~~START~~
+text
+ht-dw-cx-az
+ev-gu-by
+~~END~~
+
+
+SELECT STRING_AGG(a+b,'-') WITHIN GROUP (ORDER BY sbid DESC) FROM string_agg_t GROUP BY g
+GO
+~~START~~
+text
+az-cx-dw-ht
+by-gu-ev
+~~END~~
+
+
+SELECT STRING_AGG(a+b,'-') WITHIN GROUP (ORDER BY id, sbid ASC) FROM string_agg_t GROUP BY g
+GO
+~~START~~
+text
+az-cx-ht-dw
+gu-by-ev
+~~END~~
+
+
+SELECT STRING_AGG(a+b,'-') WITHIN GROUP (ORDER BY id, sbid DESC) FROM string_agg_t GROUP BY g
```

Comment:
Let's have one or two tests where we have ordering on string column within group

Reply:
Also let's try to have somewhat complex expression in WITHIN GROUP clause

Reply:
Done